sdks/java/core/src/main/java/org/apache/beam/sdk/runners/TransformTreeNode.java

```diff
@@ -180,17 +202,10 @@ public POutput getOutput() {
     return output;
   }
 
-  /**
-   * Returns the transform outputs, in expanded form.
-   */
-  public Collection<? extends PValue> getExpandedOutputs() {
-    if (output != null) {
-      return output.expand();
-    } else {
-      return Collections.emptyList();
-    }
+  AppliedPTransform<?, ?, ?> toAppliedPTransform() {
```

Comment:
This is linked to `recordAsOutput` + `getProducingTransformInternal` and probably will be removed with those.

Given that it's package private, I'm happy to, at least for now, leave it as an implementation detail of `TransformHeirarchy`